README_extending_docker.md

```diff
@@ -94,3 +94,21 @@ Lets consider two scenarios as starting point for extending Awesome docker
 ### How to create Custom Actions
 
 You can refer to [this link](https://docs.unskript.com/unskript-product-documentation/actions/create-custom-actions) on how to create custom Action
+
+
+## How to specify values for variables used in checks
+
+You can sepcify the variables that are used in the Checks in the Global file `unskript_config.yaml` You can see an example
```

Comment:
Reading this for the first time, doesnt tell me what i am doing to do here. Can we think from a person looking at this readme for the first time. Explain checks have arguments, so you need a way to specify. Also, how if the argument name is same across checks, you need to specify once.